microsetta_admin/server.py

```diff
@@ -135,10 +135,18 @@ def new_project():
     elif request.method == 'POST':
         project_name = request.form['project_name']
         is_microsetta = request.form.get('is_microsetta', 'No') == 'Yes'
+        bank_samples = request.form.get('bank_samples') == 'on'
```

Comment:
Would it be possible for the checkbox test to be consistent with `is_microsetta`? I don't have a preference for which approach is taken, but it would be nice that they're the same (unless there is a reason they have to differ)

Reply:
Done :)